### ISSUE TITLE
Analysis annotations overflow on edge moves

### ISSUE THREAD

**Issue by ZTL-UwU** [lichess-org/mobile#485]
### What is the problem?

The annotations in game analysis board will overflow when the move is made on the edge of the board.

![Screenshot_2024-01-18-08-23-24-745_org lichess mobileV2](https://github.com/lichess-org/mobile/assets/49156174/391166fc-07cd-4a1d-8f32-16a064afadd6)

### Steps to reproduce the bug

Go to a game with analysis and with a blunder / mistake / inaccuracy made on the edge of the board.
e.g. https://lichess.org/rPWvl5Rc move 23

### What did you expect to happen?

The annotation is shown on top of the engine gauge.

### What happened instead?

The annotation is blocked by the engine gauge.

### Device Info

Redmi K40S

### Operating System Version

Android 13

### Additional information

This is low priority as it doesn't harm the experience much.

**Comment by veloce** [comment]
The fix is "probably" simple though.
